packages/@aws-cdk/cx-api/lib/cloudformation-artifact.ts

```diff
@@ -54,6 +103,34 @@ export class CloudFormationStackArtifact extends CloudArtifact {
    */
   public readonly environment: Environment;
 
+  /**
```

Comment:
Shouldn't all this be in `cloud-assembly-schema`?

Reply:
@eladb @rix0rrr yeah, this relates to what i mentioned here: https://github.com/aws/aws-cdk/pull/6366/files#r412906367.

We don't have a schema entry for these types, only classes that read an `any` properties object and extract the expected properties. Its `MetadataEntry` all over again, and actually now that I think of it, should have been part of the work on `cloud-assembly-schema`, it was just an oversight.

Reply:
I added the property artifact definitions to the cloud-assembly-schema.